centos-ci/jobs/gluster-kubernetes.yml

```diff
@@ -0,0 +1,44 @@
+- job:
+    name: gluster_gluster-kubernetes
+    node: gluster
+    description: Run the comples test from gluster-kubernetes. Tests are
+      triggered for a pull-request after one of the repository admins approves
+      it (with standard Jenkins GitHub commands).
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: https://github.com/gluster/glusterfs-patch-acceptance-tests.git
+        branches:
+        - origin/master
+
+    properties:
+    - github:
+        url: https://github.com/gluster/gluster-kubernetes/
+
+    triggers:
+    - github-pull-request:
+        admin-list:
+        - jarrpa
+        - obnoxxx
+        white-list:
+        - jarrpa
+        - obnoxxx
+
+    builders:
+    - shell: !include-raw: ../scripts/get-node.sh
+    - shell: centos-ci/scripts/bootstrap.sh gluster-kubernetes.sh
```

Comment:
You need add a `"ghprbPullId=$ghprbPullId"` here so that it actually is passed into the script. It's a weirdness because all Jenkins variables are available to the main script. Not to the script called from boostrap unless we explicitly pass it.

Reply:
Right, that is something I missed. Thanks!